pkg/questionModel/multiSelectList.go

```diff
@@ -46,13 +46,14 @@ func (m *MultiSelectList) InitializeModel(input *QuestionInput) {
 			if index == len(items)-1 {
 				return fmt.Sprintf(xLargeLeftPadding.Render("\n[ %s ]"), blurred.Render(s))
 			}
-			return mediumLeftPadding.Render(fmt.Sprintf("%s %s", m.getCheckBox(index), s))
+			return focusTableItemRows(fmt.Sprintf("%s %s", m.getCheckBox(index), s), xLargeLeftPadding, noStyle, mediumLeftPadding)
```

Comment:
Build is failing because `focusTableItemRows` does not exist. Did it get renamed or is it otherwise missing from this diff?